Language/Haskell/GhcMod/Find.hs

```diff
@@ -120,26 +119,23 @@ loadSymbolDb' = do
         return db
   doLoad =<< liftIO (doesFileExist cache)
 
--- | Loading a file and creates 'SymbolDb'.
 loadSymbolDb :: IOish m => GhcModT m SymbolDb
 loadSymbolDb = do
-  ghcMod <- liftIO ghcModExecutable
-  readProc <- gmReadProcess'
-  out <- liftIO $ readProc ghcMod ["--verbose", "error", "dumpsym"] ""
-  return $!! decode out
+  ts <- liftIO getCurrentModTime
+  st <- runGmPkgGhc getGlobalSymbolTable
```

Comment:
This is no good, there is a reason we do this in another process (memory usage). Please revert.